openslides/users/views.py

```diff
@@ -447,7 +448,14 @@ def perform_create(self, serializer):
         Customized method to inject the request.user into serializer's save
         method so that the request.user can be saved into the model field.
         """
-        serializer.save(user=self.request.user)
+        try:
+            serializer.save(user=self.request.user)
+        except IntegrityError:
```

Comment:
Is it possible to to use this globla IntegrityError check but a more concret check whether the object exists in the database?

Reply:
This can be done, but you have to carefully put this into one transaction. The integrity error should be clear enough, because the only integrity, that can be broken is the unique user.